src/Tags/Base.php

```diff
@@ -264,4 +264,24 @@ protected function hiddenFieldForSelect($options) {
             [ 'type' => 'hidden', 'value' => '']
         ));
     }
+
+    protected function getOldInput() {
+        $index = $this->objectName;
```

Comment:
Is this ok for simply nested attrs like `person.address.province`? Won't `objectname` in that case just be `address`?